src/connection/connection_defs.h

```diff
@@ -418,6 +418,22 @@ struct css_queue_entry
 #if defined(SERVER_MODE)
 struct session_state;
 #endif
+
+#define MAX_LEN_CONNECTION_URL 512
+/*
+ * This data structure is the connection interface for dblink
+ */
+typedef struct dblink_conn_entry DBLINK_CONN_ENTRY;
+struct dblink_conn_entry
+{
+  int conn_handle;
+  char conn_url[MAX_LEN_CONNECTION_URL + 1];
+  char user_name[DB_MAX_USER_LENGTH + 1];
+  char *password[DB_MAX_PASSWORD_LENGTH + 1];
```

Comment:
```suggestion
  char password[DB_MAX_PASSWORD_LENGTH + 1];
```

`char *`가 아니고 `char` 아닌가요?

Reply:
실수가 있었습니다. ^^